### ISSUE TITLE
Scrapbot

### ISSUE THREAD

**Issue by RandomXD** [dalaranwow/dalaran-wow#5910]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The issue with scrapbot is that it can be attacked and im pretty sure it shouldn't be attackable. On a previous server I played long time ago I read in their bugtracker about fixing it so it can't be attacked since there was the same bug, also when I check comments about this I can't find anywhere anyone saying that you should be able to attack it.
This can also be used to proc trinkets as melee/hunt by attacking the scrapbot then equiping your normal trinket before boss fights
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. make a scrapbot with engi
2. summon it
3. attack it

**Include proofs for this behaviour**
![Screenshot_1](https://user-images.githubusercontent.com/36212275/87066717-ae283e80-c21b-11ea-97aa-c3c5dc5b4c7d.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
item ID 29561
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
